scripts/setLinks.py

```diff
@@ -0,0 +1,16 @@
+#!/bin/env python
```

Comment:
any reason this was done as a python wrapper around system calls, rather than just a `*sh` shell script?

Reply:
I believe this is an older script made by @cbravo135.  I'm just including it here because it provides a useful feature for setting up the per link config files.

Although this configuration schema is not a good one and should be replaced with a DB interaction rather than text files on the card.  I wouldn't want to invest too much time in this.  Rather than rewriting this in `sh` would rather just spend time writing the DB interaction configuration method.